### ISSUE TITLE
[MDEPLOY-165] deploy-file ignores classifier

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-deploy-plugin#274]
**[Gili](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cowwoc)** opened **[MDEPLOY-165](https://issues.apache.org/jira/browse/MDEPLOY-165?redirect=false)** and commented

MDEPLOY-19's implementation is incomplete. If a user specifies a `classifier` then `generatePom` fails to take the classifier into consideration and uploads a POM file that omits the classifier.

The fix is simple:

Add the following line to `DeployFileMojo.generateModel()`:

`model.setClassifier( classifier );`

---

**Affects:** 2.7

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

> model.setClassifier( classifier );

There's no such method.

**Comment by jira-importer** [comment]
**[Gili](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cowwoc)** commented

Sorry. I discovered that (no such method) when I tried formulating a patch. I misunderstood what was going on on my binary server and incorrectly assumed that the POM should contain a \<classifier> field. As you mentioned, there is no such field.

I'm closing this issue as "not a bug". Sorry for the false alarm.
